german/Leichen/Leichen.md

```diff
@@ -5,4 +5,15 @@ werfen die beiden Leichen aus dem Boot.
 
 Sie schnaufen einmal kräftig durch während ihnen tausend Gedanken durch den Kopf gehen...
 
-Schnell, aber unbemerkt versuchen Sie an Land zu kommen, aber es scheint nicht näher zu kommen....
\ No newline at end of file
+Schnell, aber unbemerkt versuchen Sie an Land zu kommen, aber es scheint nicht näher zu kommen....
+
+Was tun Sie?
+
+Ich halte nach etwas [Ausschau](Ausschau/Ausschau.md), an dass ich mich klammern könnte.
+
+All meine Kraft geht in meine zunehmend panischen [Schwimmbewegungen](Schwimmbewegungen/Schwimmbewegungen.md). Ich versuche mit letzter Kraft den Strand zu erreichen.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.